src/test/java/world/inetum/realdolmen/inetumrealJobs/controllers/AuthenticationControllerTest.java

```diff
@@ -65,4 +70,23 @@ void loginWithExistingCredentialsReturning401() throws Exception {
         assertEquals(expectedMessage, actualMessage);
     }
 
+    @Test
+    void when_UserRegisterWithRoleSelected_VerifyAcceptanceCriteria() throws Exception {
+        SignupRequest request = new SignupRequest("user1", ERole.ROLE_JOB_SEEKER, "male", "reda",
+                "saada", new Date(), "rue", "11", "1", "brussels", "1000",
+                "belgium", "+3223456789", "", "1234");
+        mockMvc.perform(post("/authentication/signUp").contentType(MediaType.APPLICATION_JSON)
+                .content(mapper.writeValueAsString(request))).andExpect(status().isOk());
```

Comment:
No backend testing that the user is created?

Reply:
Fixed